src/js/components/TextArea/StyledTextArea.js

```diff
@@ -1,6 +1,11 @@
 import styled, { css } from 'styled-components';
 
-import { focusStyle, inputStyle, placeholderStyle } from '../../utils';
+import {
+  focusStyle,
+  inputStyle,
+  placeholderStyle,
+  disabledStyle,
```

Comment:
can we please move `disabledStyle,` to line 4 and maintain the right order.

Reply:
yeap. also did for input and button